docker/intel/Dockerfile-18

```diff
@@ -30,33 +30,23 @@ COPY intel.cfg /tmp/
 RUN cd /tmp \
  && mkdir intel \
  && cd intel \
- && curl -sL http://registrationcenter-download.intel.com/akdlm/irc_nas/tec/11541/parallel_studio_xe_2017_update4_composer_edition.tgz | tar --strip-components=1 -xz \
+ && curl -sL http://registrationcenter-download.intel.com/akdlm/irc_nas/tec/13717/parallel_studio_xe_2018_update4_cluster_edition_online.tgz | tar --strip-components=1 -xz \
```

Comment:
Please use composer edition, it is a smaller download.

Reply:
That only happens once, so I'll keep it for now